apisix/plugins/api-breaker.lua

```diff
@@ -158,6 +165,13 @@ function _M.access(conf, ctx)
 
     -- breaker
     if lasttime + breaker_time >= ngx.time() then
+        if conf.break_response_body then
+            core.response.clear_header_as_body_modified()
```

Comment:
We don't need this if the body is not from the upstream

Reply:
removed this line(```core.response.clear_header_as_body_modified()```) in the new commit